sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -221,7 +221,7 @@ def _from_generated(cls, key_value):
         filters = None
         try:
             temp = json.loads(key_value.value)  # type: ignore
-            if isinstance(temp, dict):
+            if isinstance(temp, Dict):
```

Comment:
One more instance -- after this, we should be good 🙂

Reply:
Updated, thanks!